docs/docs/forms.md

```diff
@@ -2,11 +2,12 @@
 id: forms
 title: Forms
 permalink: docs/forms.html
-prev: state-and-lifecycle.html
-next: lifting-state-up.html
-redirect_from: "/tips/controlled-input-null-value.html"
+prev: 
```

Comment:
Why were these removed? Every page in the "Basic Guides" has this links.

Compare Lists and Keys (has these links):

<img width="690" alt="screen shot 2016-10-25 at 12 06 15" src="https://cloud.githubusercontent.com/assets/810438/19683749/7751e12c-9aab-11e6-93f8-5c9b489f5f00.png">

and Forms after this change (doesn't have these links anymore, why?):

<img width="726" alt="screen shot 2016-10-25 at 12 06 52" src="https://cloud.githubusercontent.com/assets/810438/19683759/8b4f5c90-9aab-11e6-81c0-fe151b933404.png">